integration/integration_test.go

```diff
@@ -2094,9 +2127,7 @@ func testMapRoles(t *testing.T, suite *integrationTestSuite) {
 	tryCreateTrustedCluster(t, aux.Process.GetAuthServer(), trustedCluster)
 	waitForTunnelConnections(t, main.Process.GetAuthServer(), clusterAux, 1)
 
-	nodePorts := helpers.NewPortSlice(3)
-	sshPort, proxyWebPort, proxySSHPort := nodePorts[0], nodePorts[1], nodePorts[2]
-	require.NoError(t, aux.StartNodeAndProxy("aux-node", sshPort, proxyWebPort, proxySSHPort))
+	sshPort, _, _ := aux.StartNodeAndProxy(t, "aux-node")
```

Comment:
Is the error ignored, or are the 2nd and 3rd arguments something else?

Reply:
Something else. Any errors are caught inside `StartNodeAndProxy()` and result in an immediately failed test.